schematic/utils/df_utils.py

```diff
@@ -57,3 +57,16 @@ def update_df(existing_df: pd.DataFrame, new_df: pd.DataFrame, idx_key: str) ->
     updated_df = updated_df[existing_df.columns]
 
     return updated_df
+
+
+def trim_commas_df(df: pd.DataFrame):
+    """Removes empty columns and empty rows from pandas dataframe (manifest data).
+
+    Args:
+        df: pandas dataframe with data from manifest file.
+
+    Returns:
+        df: cleaned-up pandas dataframe.
+    """
+    return df.dropna(how='all', axis=1) \
+             .dropna(how='all', axis=0)
```

Comment:
It's a bummer they dropped the support to do both of these at once.  In the past you could pass a tuple for the axis.

Reply:
agreed; not sure if it was for readability or to comply with general refactor of interfaces to lower level routines.